elements/pfe-navigation/README.md

```diff
@@ -66,6 +66,11 @@ All slots are optional and can be left off if not needed. Slots prefixed with `m
 - `pfe-menu-label`: Translation for the Menu label on the main navigation dropdown.
 - `pfe-full-width`: Allows the navigation to span the full width of the viewport.
 
+### CSS Variables
+
+ - `--pfe-navigation--Height--actual`: when pfe-navigation initializes, it will create a global (on `body`) CSS variable that contains the height of the pfe-navigation item.  The primary uses for this is calculating the offset for anchor links, and for positioning a sticky sub-header below the pfe-navigation.  Note that multiple pfe-navigation elements will write the same variable, unless...
```

Comment:
@chrisdo1 @castastrophe @starryeyez024 - Is someone going to finish this sentence?

![b0f76f61e0b8ee52a5e77da9371b5c93--the-lorax-sweet-tattoos](https://user-images.githubusercontent.com/330256/74740363-b65dee00-5228-11ea-9c2c-d23bcd53292f.jpg)

Reply:
The bullet below addresses it.

Reply:
LOL I thought the same thing when I read it, it took me a moment to realize the nested bullet answered the question. I think maybe a colon instead of an ellipsis would read better?

Reply:
changed this to a colon.